src/metrics/mongoose_metrics_probe_cets.erl

```diff
@@ -45,20 +42,27 @@ format_probe_cets(#{available_nodes := AvNodes,
       <<"conflict_tables">> => ConTabs}.
 
 all_zeros() ->
-    #{<<"available_nodes">> => 0,
-      <<"unavailable_nodes">> => 0,
-      <<"joined_nodes">> => 0,
-      <<"discovered_nodes">> => 0,
-      <<"discovery_works">> => 0,
-      <<"remote_nodes_without_disco">> => 0,
-      <<"remote_nodes_with_unknown_tables">> => 0,
-      <<"remote_unknown_tables">> => 0,
-      <<"remote_nodes_with_missing_tables">> => 0,
-      <<"remote_missing_tables">> => 0,
-      <<"conflict_nodes">> => 0,
-      <<"conflict_tables">> => 0}.
+    #{available_nodes => 0,
+      unavailable_nodes => 0,
+      joined_nodes => 0,
+      discovered_nodes => 0,
+      discovery_works => 0,
+      remote_nodes_without_disco => 0,
+      remote_nodes_with_unknown_tables => 0,
+      remote_unknown_tables => 0,
+      remote_nodes_with_missing_tables => 0,
+      remote_missing_tables => 0,
+      conflict_nodes => 0,
+      conflict_tables => 0}.
 
-datapoints() ->
+-spec instrumentation(_) -> [mongoose_instrument:spec()].
+instrumentation(_) ->
```

Comment:
This function shouldn't take any argument, because the intention was to accept host types.